owncloudApp/src/main/java/com/owncloud/android/authentication/BiometricManager.java

```diff
@@ -145,6 +136,11 @@ public boolean hasEnrolledBiometric() {
      * USE WITH CARE
      */
     public void bayPassUnlockOnce() {
-        setUnlockTimestamp();
+        int timeout = LockTimeout.Companion.fromStringToMilliseconds(preferencesProvider.getString(LOCK_TIMEOUT, LockTimeout.IMMEDIATELY.name()));
+        long lastUnlockTimestamp = preferencesProvider.getLong(LAST_UNLOCK_TIMESTAMP, 0);
+        if (System.currentTimeMillis() - lastUnlockTimestamp > timeout) {
```

Comment:
I have seen those 3 lines somewhere else 🤔  Could we use a common function to check this? Maybe with a descriptive name 👍

Reply:
Well in fact not only those 3 lines, the whole function 😄 . So, I don't have to think about a new descriptive name, `bayPassUnlockOnce` has been moved to `SecurityUtils.kt` 😛